### PR TITLE
fixes ownProps not being passed to mapToProps inside connect HOC 

### PR DESCRIPTION
**Pull request by malbernaz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "lint": "tslint src/**/*.ts src/**/*.tsx",
     "test": "jest",
     "format": "prettier --write --no-semi \"src/**/*.ts\" \"src/**/*.tsx\"",
-    "check": "npm run compile && npm run format && npm run lint && npm run test",
+    "check":
+      "npm run compile && npm run format && npm run lint && npm run test",
```
Comment (matheusml) [inline package.json]: What happened here?

Comment (malbernaz) [reply package.json]: prettier...

**Comment by matheusml** [comment]
Is this also happening for preact bindings, @malbernaz?

**Comment by malbernaz** [comment]
no it happens just on react's connect HOC.

**Comment by malbernaz** [comment]
Thanks, man! Had a bug in production concerning this bug...

**Pull request merged**